.circleci/config.yml

```diff
@@ -509,39 +518,50 @@ workflows:
             cd tests/integration/cli
             ./cli-tests.sh ci_3
           requires:
-            - redshift-itest
+            - redshift-s3-itest
+          filters:
+            tags:
+              only: /v\d+\.\d+\.\d+(-[\w]+)?/
+      - integration_test:
+          name: redshift-s3-itest
+          extras: '[redshift-binary,itest]'
+          python_version: "3.6"
+          db_name: demo-itest
+#          requires:  # TODO
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker